content/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners.md

```diff
@@ -39,6 +39,18 @@ Each CODEOWNERS file assigns the code owners for a single branch in the reposito
 
 For code owners to receive review requests, the CODEOWNERS file must be on the base branch of the pull request. For example, if you assign `@octocat` as the code owner for *.js* files on the `gh-pages` branch of your repository, `@octocat` will receive review requests when a pull request with changes to *.js* files is opened between the head branch and `gh-pages`.
 
+## CODEOWNERS file size
```

Comment:
I think that the new section is repeated twice and one of them needs to be ✂️

Reply:
Oh my you are right 😅 . Thanks for catching that!